util/build/script/src/env.rs

```diff
@@ -320,9 +320,8 @@ impl Environment {
                     if value.is_empty() {
                         None
                     } else {
-                        Some(value.parse().map_err(|e| {
-                            EnvironmentError::ParseInt(ENV_CARGO_PKG_VERSION_PRE.to_owned(), e)
-                        })?)
+                        // FIXME: 0.1.3-pre1 cannot be parsed due to "not a digit"
```

Comment:
@jcape - this is erroring currently - I'm ham-fisting it to a string, but I wonder if I'm missing something here.